### ISSUE TITLE
XLM-Roberta mask filling error

### ISSUE THREAD

**Issue by AzamatSultonov** [huggingface/transformers#2741]
# XLM-Roberta mask token filling error

Hi! I am trying to use XLM-Roberta for Masked LM task, but the error occurs when the model fills masked token in a test sentence.

**The code is:**

```
config.model_name = 'xlm-roberta-base'

tokenizer: tr.XLMRobertaTokenizer = tr.XLMRobertaTokenizer.from_pretrained(config.model_name)
model: tr.XLMRobertaForMaskedLM = tr.XLMRobertaForMaskedLM.from_pretrained(config.model_name)

input_ids = tokenizer.encode_plus("I want to <mask> New York!", 
                                               max_length=config.max_length)['input_ids']
x = np.full((config.max_length), fill_value=tokenizer.pad_token_id)
attn = np.zeros_like(x)
for i, tok in enumerate(input_ids):
    x[i] = tok
    attn[i] = 1
x = torch.tensor(x).unsqueeze(0).to(device)
attn = torch.tensor(attn).unsqueeze(0).to(device)

outputs = model(x, attention_mask=attn, masked_lm_labels=x)
```
**The error is**

```
RuntimeError: cublas runtime error : library not initialized at ../aten/src/THC/THCGeneral.cpp:216
```

When I try Albert for similar task everything works fine, but the Roberta family doesn't.
Could you please help with this issue?

**Comment by AzamatSultonov** [comment]
Found solution in [#2509](https://github.com/huggingface/transformers/pull/2509).

**Comment by LysandreJik** [comment]
Hi, indeed this is an error. This will be fixed once #3198 is merged.
